src/ui/profile_preferences.ui

```diff
@@ -1341,124 +1341,90 @@
        <string>Mapper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_24">
-       <item row="1" column="0">
-        <widget class="QGroupBox" name="downloadMapOptions">
+       <item row="4" column="0">
```

Comment:
Without keeping (just) the `QGridLayout` entries (which can get messed about if an item inside is edited) in the same order the git commit on the `.ui` file is much larger and it is almost impossible to decipher what is being changed... 😒